Authenticator.spec.ts

```diff
@@ -0,0 +1,154 @@
+			name: { first: "", last: "" },
+			email: "",
+			permissions: flagly.Flags.stringify(permissions),
+		})
+		const authenticated = await authenticator.authenticate(token ?? "", {
+			[organization ?? "*"]: { cards: { view: true } },
+		})
+		const authenticatedNoRealm = await authenticator.authenticate(token ?? "", {
+			["*"]: { cards: { view: true } },
+		})
+		expect(authenticated).toBeTruthy()
+		expect(authenticatedNoRealm).toBeTruthy()
+	})
+	it("permissions example all organizations", async () => {
+		const organization: string | undefined = "asdf"
+		const permissions: pax2pay.Key.Permissions = { "*": { cards: true } }
+		const token = await issuer.sign({
+			name: { first: "", last: "" },
+			email: "",
+			permissions: flagly.Flags.stringify(permissions),
+		})
+		const authenticated = await authenticator.authenticate(token ?? "", {
+			[organization ?? "*"]: { cards: { view: true } },
+		})
+		const authenticatedNoRealm = await authenticator.authenticate(token ?? "", {
+			["*"]: { cards: { view: true } },
+		})
+		expect(authenticated).toBeTruthy()
+		expect(authenticatedNoRealm).toBeTruthy()
+	})
+	it("permissions example one realm", async () => {
+		const realm: pax2pay.Realm | undefined = undefined
+		const permissions: pax2pay.Key.Permissions = { test: { cards: true } }
+		const token = await issuer.sign({
+			name: { first: "", last: "" },
+			email: "",
+			permissions: flagly.Flags.stringify(permissions),
+		})
+		const authenticated = await authenticator.authenticate(token ?? "", {
+			[realm ?? "*"]: { cards: { view: true } },
+		})
+		expect(authenticated).toBeTruthy()
+	})
+	it("permissions example several realms", async () => {
+		const realm: pax2pay.Realm | undefined = "test"
+		const permissions: pax2pay.Key.Permissions = { test: { cards: true }, testUK: { cards: true } }
+		const token = await issuer.sign({
+			name: { first: "", last: "" },
+			email: "",
+			permissions: flagly.Flags.stringify(permissions),
+		})
+		const authenticated = await authenticator.authenticate(token ?? "", {
+			[realm ?? "*"]: { cards: { view: true } },
+		})
+		const authenticatedNoRealm = await authenticator.authenticate(token ?? "", {
+			["*"]: { cards: { view: true } },
+		})
+		expect(authenticated).toBeTruthy()
+		expect(authenticatedNoRealm).toBeTruthy()
+	})
+	it("permissions example all realms", async () => {
+		const realm: pax2pay.Realm | undefined = "test"
+		const permissions: pax2pay.Key.Permissions = { "*": { cards: true } }
+		const token = await issuer.sign({
+			name: { first: "", last: "" },
+			email: "",
+			permissions: flagly.Flags.stringify(permissions),
+		})
+		const authenticated = await authenticator.authenticate(token ?? "", {
+			[realm ?? "*"]: { cards: { view: true } },
+		})
+		const authenticatedNoRealm = await authenticator.authenticate(token ?? "", {
+			["*"]: { cards: { view: true } },
+		})
+		expect(authenticated).toBeTruthy()
+		expect(authenticatedNoRealm).toBeTruthy()
+	})
+})
```

Comment:
This is just the same code over and over. Also most comments apply to several rows but I don't want to repeat myself.